tests/unit-test/test_scylladb.c

```diff
@@ -199,8 +199,8 @@ void test_db_identity_table(void) {
 }
 #endif  // DB_ENABLE
 
-int main(int argc, char** argv) {
 #ifdef DB_ENABLE
+int main(int argc, char** argv) {
```

Comment:
Don't do that. Use the following instead:
```c
int main(int argc, char** argv) {
#if !defined(DB_ENABLE)
  return 0;
#endif
   ... (original code) ...
}
```

Reply:
I modify the Bazel rule as well for this changement.